internal/output/output.go

```diff
@@ -36,13 +36,45 @@ type resources struct {
 	Items []resource `json:"items"`
 }
 
-// CmdOutput returns a command's response as a string formatted based on the user's requested type.
-func CmdOutput(outputKind string, outputter OutputterFn) (string, error) {
-	o, err := outputter.New()
+// resourcesBare is for responses that return a list of resources at the top level of the response,
+// not as a value of an "items" property.
+type resourcesBare []resource
+
+func CmdOutputResource(outputKind string, input []byte, fn PlaintextOutputFn) (string, error) {
+	var r resource
+	err := json.Unmarshal(input, &r)
 	if err != nil {
 		return "", err
 	}
 
+	return outputFromKind(outputKind, SingularOutputter{
+		outputFn:     fn,
+		resource:     r,
+		resourceJSON: input,
+	})
+}
+
+func CmdOutputResources(outputKind string, input []byte, fn PlaintextOutputFn) (string, error) {
+	var r resources
+	err := json.Unmarshal(input, &r)
+	if err != nil {
+		// sometimes a response doesn't include each item in an "items" property
```

Comment:
It'd be nice not to have to do this, but turns out our API is a little inconsistent...